solutions/olof/stone.js

```diff
@@ -0,0 +1,114 @@
+let valElements = document.querySelectorAll(".val");
+
+const stenp1 = document.getElementById("stenp1");
+const saxp1 = document.getElementById("saxp1");
+const påsep1 = document.getElementById("påsep1");
+
+const stendata = document.getElementById("stenData");
+const saxdata = document.getElementById("saxData");
+const påsedata = document.getElementById("påseData");
+
+const playAgainButton = document.querySelector("#playAgainButton");
+const getLostElements = document.querySelectorAll(".getLost");
+
+let p1Val = undefined;
+let dataVal = undefined;
+
+
+stenp1.addEventListener('click', function () {
+    p1Val = "sten";
+    stenp1.style.backgroundColor = "pink";
+    setTimeout(() => {
+        dataValAndCheckResult();
+    }, 1432);
```

Comment:
`1432` is a repeating constant. I suggest that you make a variable `const waitingTime = 1432` and use it instead.